kernel/interrupt.c

```diff
@@ -36,20 +37,24 @@ static const char * exception_names[] = {
 
 static void unknown_exception( int i, int code )
 {
-	unsigned vaddr, paddr;
+	unsigned vaddr, paddr, esp;
```

Comment:
Docs nit: most of these are called out in code further on, but could we just describe them here?